iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/TerminalRefXml.java

```diff
@@ -24,21 +25,25 @@ public static void writeTerminalRef(Terminal t, NetworkXmlWriterContext context,
     }
 
     public static void writeTerminalRef(Terminal t, NetworkXmlWriterContext context, String namespace, String elementName) throws XMLStreamException {
+        writeTerminalRef(t, context, namespace, elementName, context.getWriter());
+    }
+
+    public static void writeTerminalRef(Terminal t, NetworkXmlWriterContext context, String namespace, String elementName, XMLStreamWriter writer) throws XMLStreamException {
```

Comment:
@MioRtia if I remerber well, there is an extension for Battery that use the old variant of this method. Could you add it to your todo list.

Reply:
Yes, this motivated this PR